libs/martech/martech.js

```diff
@@ -118,14 +118,14 @@ export const getTargetPersonalization = async () => {
   const responseStart = Date.now();
   window.addEventListener(ALLOY_SEND_EVENT, () => {
     const responseTime = calculateResponseTime(responseStart);
-    window.lana.log(`target response time: ${responseTime}`, { tags: 'errorType=info,module=martech' });
+    window.lana.log(`target response time: ${responseTime}`, { tags: 'module-martech', errorType: 'i' });
```

Comment:
Where do you have this errorType from? The type is correct as far as I can see according to the initial PR where we introduced the feature https://github.com/adobecom/milo/pull/1463 
// cc @robert-bogos

Reply:
I have used the Lana documentation to set the errorType option to 'i'.
Fyr: https://github.com/adobecom/milo/blob/main/libs/utils/lana.md

Reply:
Theat file is little older than the linked PR and wasn't updated it seems. Mind changing that?

Reply:
How do i set the errorType of a log to 'info' according to the initial PR?

Reply:
There's a lot of examples in the PR 🤔 https://github.com/adobecom/milo/pull/1463/files

Reply:
My understanding was setting the errorType within tags as "info" would just help to filter logs in Splunk and doesn't change the log type to "info", correct me if i'm wrong.

Reply:
Yes, the errorType purpose was to help filter logs in Splunk

Reply:
After more discussions on the matter, we decided that the source of truth will be this [lana.md](https://github.com/adobecom/milo/blob/main/libs/utils/lana.md). [My initial PR](https://github.com/adobecom/milo/pull/1463/files) for this is not relevant anymore.